cmd/security-bootstrap-redis/README.md

```diff
@@ -0,0 +1,21 @@
+# EdgeX Foundry Security Service - Security Bootstrap Redis
+
+[![license](https://img.shields.io/badge/license-Apache%20v2.0-blue.svg)](LICENSE)
+
+## Summary
+
+Implements a long running service to read the Redis password from Vault and set it in Redis.
+
+## Explanation
+
+In the Geneva release, this code was liberally borrowed from [edgexfoundry/docker-edgex-mongo](https://github.com/edgexfoundry/docker-edgex-mongo), tightly coupled with security-secrets-setup, and used a shared file to pass the Redis password to Redis. For the Hanoi release, the goal is to conform to the [ADR for secret creation and distribution](https://docs.edgexfoundry.org/1.2/design/adr/security/0008-Secret-Creation-and-Distribution/) and support configuration overrides via go-mod-boostrap.
+
+The service is organized in the Docker compose file to run after security-secretstore-setup and Redis are started. This isn't guaranteed by Docker so the security-bootstrap-redis will keep retrying until the retry timer expires. The service starts by reading the Redis password from the vault, creates a connection to Redis (which when it starts does not require authentication), sets the password obtained from the vault, and finally uses the formal database bootstrap handler to verify the password is set as expected.
```

Comment:
In the current implementation, if the Redis password is set and you start security-bootstrap-redis, it will exit because it cannot get a clear connection to Redis.

Are you asking me to document this in the README?

Reply:
I will cover this scenario in the code and document it in the README.

Reply:
So would this prevent an upgrade from Geneva to Redis? Was there a reason to change the keys/paths that are used to store the credentials in Vault (e.g. `redis` --> `bootstrap-redis`, `redis5` --> `redisdb`)?

Reply:
I realized my first implementation didn't conform well to the pattern created in the original design so with this iteration I brought it in line. To upgrade you need to restart security-secretstore-setup and Redis, and staart security-bootstrap-redis.